### PR TITLE
Documenting the Mail setup on production

### PR DESCRIPTION
**Pull request by carolyncole**
Also documnenting mailcatcher for development
refs #687

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +83,20 @@ To See mail that has been sent on the staging server you must ssh tunnel into th
...
+Mailcatcher is a gem that can also be installed locally.  See the [mailcatcher documentation](https://mailcatcher.me/) for how to run it on your machine.
+
+Once mailcatcher is running set your config/environments/development.rb to include:
+```
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: "localhost",
+    port: 1025
+  }
+```
```
Comment (mccalluc) [inline README.md:99]: Is there an argument against just making mailcatcher part of the default dev environment? I know that once I've made changes to my local config, unless the file is gitignored, I'm probably going to check it in at some point by mistake. It's hard to break the `git add .` habit.

**Review by mccalluc — COMMENTED** [review]
This is really helpful, but I wonder if it could just be the default?

File: README.md (hunk 1 above)
Comment (jrgriffiniii) [reply README.md:99]: This was discussed during the check-in meeting, and might it be possible for me to please issue a pull request setting this as the default for the application instead? I apologize, I can issue the PR proposing these changes momentarily.

Comment (jrgriffiniii) [reply README.md:99]: Please see https://github.com/pulibrary/pdc_describe/pull/872

**Pull request merged**